requirements-lint.in

```diff
@@ -0,0 +1,3 @@
+flake8<6
```

Comment:
Can we add a comment here noting why it is pinned.  Maybe this?
```suggestion
# Pinned because `flake8-copyright==0.2.3` is incompatible with `flake8>=6`.  Can unpin this
# when https://github.com/savoirfairelinux/flake8-copyright/pull/20 or a similar fix is released
flake8<6
```